### ISSUE TITLE
ComboBox crashes when Y = Pos.Fill() is used

### ISSUE THREAD

**Issue by tig** [migueldeicaza/gui.cs#613]
@fergusonr, of course, I discover this right after I accept the PR. ;-)

change the Scenario to this:

```csharp
			var comboBox = new ComboBox() {
				X = Pos.Right(listview) + 1 , 
				Y = Pos.Bottom (lbListView) + 1,
				Height = Dim.Fill(),
				Width = 30
			};
			comboBox.SetSource (items);
```

and you'll see. 

It also crashes if Dim.Fill is used on `Width`.

**Comment by fergusonr** [comment]
Ah, assign to me please

**Comment by tig** [comment]
This breaks common Computed layout scenarios. You will need to revisit your design.

**Comment by fergusonr** [comment]
Fixed on PR #624
